modules/porous_flow/include/materials/PorousFlowMaterialPermeabilityKozenyCarman.h

```diff
@@ -0,0 +1,92 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef POROUSFLOWMATERIALPERMEABILITYKOZENYCARMAN_H
+#define POROUSFLOWMATERIALPERMEABILITYKOZENYCARMAN_H
+
+#include "DerivativeMaterialInterface.h"
+#include "Material.h"
+
+#include "PorousFlowDictator.h"
+
+//Forward Declarations
+class PorousFlowMaterialPermeabilityKozenyCarman;
```

Comment:
Just call it PorousFlowPermeabilityKozenyCarman (ie, remove the "Material").  We initially had PorousFlowMaterialXXXX everywhere but then decided it was just too long and we remove the "Material".  Perhaps there are still some classes with "Material" in them, ooops, but don't create a new one.